### PR TITLE
Updated image links to have newer format.

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
New format: Panel share embed

### CODE REVIEW

File: docs/sources/sharing/share-dashboard.md [hunk 1]
```diff
@@ -33,17 +32,17 @@ A dashboard snapshot shares an interactive dashboard publicly. Grafana strips se
...
 provided by Grafana Labs that allows you to publish dashboard snapshots to an external Grafana instance. The same rules still apply: anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
 
-{{< docs-imagebox img="/img/docs/sharing/Share_Dashboard_Snapshot.png" max-width="700px" >}}
+![Dashboard share snapshot](/img/docs/sharing/share-dashboard-snapshot-7-3.png)
 
 To publish a snapshot:
 
 1. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
-1. Copy the snapshot link, and share it either within your organization or publicly on the web.
+2. Copy the snapshot link, and share it either within your organization or publicly on the web.
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-dashboard.md:40]:
```suggestion
1. Copy the snapshot link, and share it either within your organization or publicly on the web.
```

File: docs/sources/sharing/share-panel.md [hunk 2]
```diff
@@ -14,8 +14,7 @@ To share a panel:
 1. Click a panel title to open the panel menu.
 1. Click **Share**. The share dialog opens and shows the Link tab.
 
-{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Direct_Link.png" max-width="700px" >}}
-
+![Panel share direct link](/img/docs/sharing/share-panel-direct-link-7-3.png)
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:17]:
```suggestion
![Panel share direct link](/img/docs/sharing/share-panel-direct-link-7-3.png)

```

File: docs/sources/sharing/share-panel.md [hunk 3]
```diff
@@ -47,21 +46,21 @@ A panel snapshot shares an interactive panel publicly. Grafana strips sensitive
...
 
-{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Snapshot.png" max-width="700px" >}}
+![Panel share snapshot](/img/docs/sharing/share-panel-snapshot-7-3.png)
 
 To publish a snapshot:
 
 1. In the Share Panel dialog, click **Snapshot** to open the tab.
-1. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
-1. Copy the snapshot link, and share it either within your organization or publicly on the web.
+2. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:54]:
```suggestion
1. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
```

File: docs/sources/sharing/share-panel.md [hunk 4]
```diff
@@ -47,21 +46,21 @@ A panel snapshot shares an interactive panel publicly. Grafana strips sensitive
...
+3. Copy the snapshot link, and share it either within your organization or publicly on the web.
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:55]:
```suggestion
1. Copy the snapshot link, and share it either within your organization or publicly on the web.
```

File: docs/sources/sharing/share-dashboard.md [hunk 5]
```diff
@@ -14,8 +14,7 @@ To share a dashboard:
 1. Go to the home page of your Grafana instance.
 1. Click on the share icon in the top navigation. The share dialog opens and shows the Link tab.
 
-{{< docs-imagebox img="/img/docs/sharing/Share_Dashboard_Direct_Link.png" max-width="700px" >}}
-
+![Dashboard share direct link](/img/docs/sharing/share-dashboard-direct-link-7-3.png)
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-dashboard.md:17]:
```suggestion
![Dashboard share direct link](/img/docs/sharing/share-dashboard-direct-link-7-3.png)

```

**Review by oddlittlebird — APPROVED** [review]
A few minor edits, then g2g

File: docs/sources/sharing/share-dashboard.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-dashboard.md:17]: What is the change here?

File: docs/sources/sharing/share-panel.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:17]: Not sure what is the change here?

File: docs/sources/sharing/share-dashboard.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-dashboard.md:40]: Done

File: docs/sources/sharing/share-panel.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:54]: Done

File: docs/sources/sharing/share-panel.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md]: Done

**Pull request merged**